### DISCUSSION TITLE
how to convert GAP source file to bin file?

### DISCUSSION THREAD

**Discussion by yatang32** [bitcraze/discussions#963] [Q&A]
I am trying to upload our code to GAP 8 in AI deck.

I successfully use cfclient to flash the boot loader and flash wifi streamer bin file.

The next question is how can I convert a src code to bin for GAP8 in AI deck.
I cannot find it anywhere in GAP8 manual. 

YT YANG

**Comment by knmcguire** [comment]
So I'm assuming you already did [the getting started tutorial](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/)?

The tutorial then refers to some examples in the bottom, and also to build the wifi-example itself. That should be a good start to try out building your own bin file: https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/simple-examples/wifi-streamer/

**Comment by yatang32** [comment]
把 souce code 轉成 bin
再上傳 用cfclient

YT YANG
